onvm/onvm_nflib/onvm_pkt_common.c

```diff
@@ -113,8 +113,10 @@ onvm_pkt_process_tx_batch(struct queue_mgr *tx_mgr, struct rte_mbuf *pkts[], uin
                 } else if (meta->action == ONVM_NF_ACTION_OUT) {
                         nf->stats.act_out++;
                         if (tx_mgr->mgr_type_t == MGR) {
+				//printf("tx_mgr->mgr_type_t == MGR\n");
                                 onvm_pkt_enqueue_port(tx_mgr, meta->destination, pkts[i]);
                         } else {
+				//printf("tx_mgr->mgr_type_t is not MGR\n");
```

Comment:
before we merge the code you'll need to go through and clean up these changes that you don't need any more.